docs/drive-replacement.md

```diff
@@ -0,0 +1,237 @@
+Drive replacement (applies for versions > v3.2.2)
+-------------
+
+These are the steps to move the volumes from one drive to another drive from the **same node**. These steps will be useful during drive replacement.
+
+(NOTE: This will just move the volume references from source to the destination drive without moving the actual data)
```

Comment:
So for non-MinIO workloads, where there is no auto-disk healing, should we have some guidance here?

"For workloads which do not support drive-level healing, you can manually migrate data from the old drive to the new drive using standard filesystem tools such as `dd` or `cp`."

Reply:
I feel its better not to comment on the "data" part. Can we just say that the "data" wont be moved but just the references.

Reply:
No objections